nameko/cli/run.py

```diff
@@ -92,12 +92,14 @@ def import_service(module_name):
     return found_services
 
 
-def setup_backdoor(runner, port):
+def setup_backdoor(runner, address, port):
     def _bad_call():
         raise RuntimeError(
             'This would kill your service, not close the backdoor. To exit, '
             'use ctrl-c.')
-    socket = eventlet.listen(('localhost', port))
+    address = address if address in {'0.0.0.0', '127.0.0.1'} else '127.0.0.1'
```

Comment:
I think it would be valid to specify other bind addresses than just `0.0.0.0` and `127.0.0.1`. Also this doesn't take into account that the bind address could support ipv6.

It may be simpler to just accept the bind address that is supplied or default to `127.0.0.1` and log a warning if the bind address is not `127.0.0.1` or `::1`.